spec/lib/api/openid_connect/protected_resource_endpoint_spec.rb

```diff
@@ -0,0 +1,85 @@
+require "spec_helper"
+describe Api::OpenidConnect::ProtectedResourceEndpoint, type: :request do
```

Comment:
Ensure a blank line between `require` and `describe` please. There are other occurences.